lib/trie_ascoltatore.js

```diff
@@ -50,11 +51,14 @@ TrieAscoltatore.prototype.publish = function (topic, message, options, done) {
 
   var cbs = this._matcher.match(topic);
 
-  for (var i = 0; i < cbs.length; i++) {
-    cbs[i](topic, message, options);
+  function clientCallback(cb, next){
+    setImmediate(function(){
+      cb(topic, message, options);
+    });
+    next();
```

Comment:
We can bring `next` inside `setImmediate` right after `cb` returned. I just felt slow client callbacks should be don't care! so that we can report back from `.publish` ASAP!